influxql/engine.go

```diff
@@ -669,54 +669,66 @@ func ReduceMean(key Key, values []interface{}, e *Emitter) {
 
 // MapMin collects the values to pass to the reducer
 func MapMin(itr Iterator, e *Emitter, tmin int64) {
-	var values []float64
+	var min *float64
```

Comment:
Wouldn't it be easier to just set min to `math.MaxFloat64` and then avoid doing the pointer stuff? Then only emit if `min != math.MaxFloat64`. Of course then you couldn't have that as the min value, but I think that's ok